pkg/vex/statement.go

```diff
@@ -102,14 +102,14 @@ func SortStatements(stmts []Statement, documentTimestamp time.Time) {
 
 		iTime := stmts[i].Timestamp
 		if iTime.IsZero() {
-			iTime = documentTimestamp
+			iTime = &documentTimestamp
```

Comment:
Should this check be `if iTime == nil { ...` now?

(and same thing for `jTime`)